src/main/java/com/kruize/service/RecommendationsService.java

```diff
@@ -117,25 +118,45 @@ private JsonObject getApplicationJson(ApplicationRecommendationsImpl application
 
         JsonObject resourcesJson = getResourceJson(applicationRecommendations, application);
 
-        applicationRecommendationJson.add("resources", resourcesJson);
-
-        return applicationRecommendationJson;
+        if (resourcesJson != null)
+        {
+            applicationRecommendationJson.add("resources", resourcesJson);
+            return applicationRecommendationJson;
+        }
+        else
+        {
+            return null;
+        }
```

Comment:
Brackets are not conforming to our coding standard.

Reply:
Fixed

Reply:
Hmm, I meant the bracket has to be on the same line as `if`. Also please avoid doing force pushes as that takes away the previous commits and the context. Force pushes should be used extremely rarely and not for code reviews during a PR approval process. Otherwise it makes it extremely difficult to review.